api/lib/domain/models/Assessment.js

```diff
@@ -9,6 +9,7 @@ const states = {
   COMPLETED: 'completed',
   STARTED: 'started',
   ABORTED: 'aborted',
+  ENDED_BY_SUPERVISOR: 'endedBySupervisor',
```

Comment:
**chipotage**
Le message de commit est "add endedBySupervisor state to an assessment" ce qui correspond à cette ligne, mais qui me semble un peu accessoire par rapport du commit.
Il me semble que le véritable objet du commit c'est l'ajout de la méthode `endBySupervisorByAssessmentId` au `assessementRepository`